copy_encrypted_ami.sh

```diff
@@ -46,7 +46,7 @@ command -v aws >/dev/null 2>&1 || die "aws cli is required but not installed. Ab
 
 
 
-while getopts ":s:d:a:l:r:k:nth" opt; do
+while getopts ":s:d:a:N:l:r:k:nth" opt; do
```

Comment:
Could you also add in the usage description in the usage function ?

Reply:
Oh, yes sorry - done